### ISSUE TITLE
Is Network booting Raspberry pi 3B+ possible

### ISSUE THREAD

**Issue by Joel-Mckay** [FOGProject/fogproject#280]
According to the documentation it suggests it is supported:
https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/net.md

My intent was to try creating a disk-less  Alpine workstation over a nfs mount:
https://www.alpinelinux.org/downloads/

1. Can FOG be easily configured to serve a disk-less ram-based OS image?
(similar to the clamav scanner utility common on some networks)

2. Is it possible to detect the distinct arm core within the normal PXE platform identifier ?
(Specifically, can the Microchip LAN7515 cause confusion identifying other PXE device CPU ids)

3. Can FOG use multiple NICs, and distinguish which image should be served on which network. Or is it limited to loading the disk-imaging utility?

Cheers,
J

**Comment by Sebastian-Roth** [comment]
@Joel-Mckay Why would you use FOG for netbooting a disk-less OS image in the first place? This can be setup without the overhead of FOG imaging solution.

FOG as of now is not able to serve on multiple NICs. Not saying this cannot be done but it definitely needs some major manual modification.

If you still think FOG is what you are aiming for then I highly recommend you ask this question in the [FOG forums](https://forums.fogproject.org/) as we have more people actively answering those kind of things there.

**Comment by Joel-Mckay** [comment]
Thanks, I will check out the forums... ;-)
I am trying to load test a SAN made out of heterogeneous hardware to help simulate some cluster coordination algorithms.  FOG covers most platforms out of the box for imaging, but I am curious if the ARM PXE hardware ids would interfere with existing infrastructures.

Being open source, FOG also seems more adaptable than an Altiris Deployment for this type of functional context sensitive network isolation.   As the ARM servers gain in popularity, I am sure more people will be seeking the same functionality in the near future.

Cheers,
J

**Comment by ghooper90** [comment]
Yes be sure to come over to the forums. We were just having a discussion about ARM process support this week. As for ARM pxe information I would be interested to knowing that too. I have some tutorials that explain how to capture this data. I'd like to take a look at the pcap of the pxe booting process to see how it differs from the x86 systems.

**Comment by Sebastian-Roth** [comment]
@Joel-Mckay Any news on this? I am wondering if you made it to the forums. We added ARM support a little while ago but it's disabled right now as it seems like no one is using it. Raise your voice in the forums if you are still keen to get this to work. Closing this for now.

Feel free to open a new issue report if you have a specific thing we should change. Be aware that ARM support is something to be addressed in the FOS Linux system which has it's [own repo](https://github.com/FOGProject/fos).
